main/js/download-scritps.js

```diff
@@ -1,5 +1,67 @@
+// Installer: ARM, x64, x86 THEN Portable: ARM, x64, x86
+const windowsLinks = [
+  "https://github.com/KnossosNET/Knossos.NET/releases/download/v0.2.0-RC10/Knossos.NET-0.2.0-RC10-arm64.ex", 
+  "https://github.com/KnossosNET/Knossos.NET/releases/download/v0.2.0-RC10/Knossos.NET-0.2.0-RC10-x64.exe", 
+  "https://github.com/KnossosNET/Knossos.NET/releases/download/v0.2.0-RC10/Knossos.NET-0.2.0-RC10-x86.exe",
+  "https://github.com/KnossosNET/Knossos.NET/releases/download/v0.2.0-RC10/Windows_arm64.zip",
+  "https://github.com/KnossosNET/Knossos.NET/releases/download/v0.2.0-RC10/Windows_x64.zip",
+  "https://github.com/KnossosNET/Knossos.NET/releases/download/v0.2.0-RC10/Windows_x86.zip"
+];
+
+const windowsVersions = [
+  "0.2.0-RC10",
+  "0.2.0-RC10",
+  "0.2.0-RC10",
+  "0.2.0-RC10",
+  "0.2.0-RC10",
+  "0.2.0-RC10"
+]
+
+// Universal, then Apple Silicon, Then Intel
+const macLinks = [
+  "https://github.com/KnossosNET/Knossos.NET/releases/download/v0.2.0-RC10/Knossos.NET-0.2.0-RC10.dmg",
+  "https://github.com/KnossosNET/Knossos.NET/releases/download/v0.2.0-RC10/MacOS_arm64.tar.gz", 
+  "https://github.com/KnossosNET/Knossos.NET/releases/download/v0.2.0-RC10/MacOS_x64.tar.gz"
+];
+
+const macVersions = [
+  "0.2.0-RC10",
+  "0.2.0-RC10",
+  "0.2.0-RC10"
+]
+
+// Appimage: aarch64, x86_64 THEN Precombiled Binaries aarch64, x86_64
+const linuxLinks = [
+  "https://github.com/KnossosNET/Knossos.NET/releases/download/v0.2.0-RC10/Knossos.NET-aarch64.AppImage", 
+  "https://github.com/KnossosNET/Knossos.NET/releases/download/v0.2.0-RC10/Knossos.NET-x86_64.AppImage",
+  "https://github.com/KnossosNET/Knossos.NET/releases/download/v0.2.0-RC10/Linux_arm64.tar.gz",
+  "https://github.com/KnossosNET/Knossos.NET/releases/download/v0.2.0-RC10/Linux_x64.tar.gz"
+];
+
+const linuxVersions = [
+  "0.2.0-RC10",
+  "0.2.0-RC10",
+  "0.2.0-RC10",
+  "0.2.0-RC10"
+]
+
+const latestManualVersion = "0.2.0-RC10";
+
 // Run at the start of the page (called from the html) with our best guess at 
 function initPage(arch){
+  populateFields();
+
+  try{
+    fetch("https://api.github.com/repos/KnossosNET/Knossos.NET/releases/latest")
+    .then((response) => response.json())
+    .then((responseJSON) => get_info(responseJSON))
+    .then(populateFields);
+  } 
+  catch (error) {
+    console.log("Fetching the most recent build from the github api failed. The error encountered was:")
+    console.error(error);
+  }
```

Comment:
Sorry for the confusion with my recommendation here, but the catch really needs to be on the promise chain as opposed to using a try-catch block. So:
```
.then(populateFields)
.catch(error => console.error(error));
```
(or however you want to handle error reporting)

Reply:
And this is why I was looking for someone who knew what they were doing.  I did not know this was a thing.